packages/webapp/pages/squads/index.tsx

```diff
@@ -15,15 +15,33 @@ import {
   SourceCard,
```

Comment:
Feel like our components always end up with lots of lines of code making it harder to read....

Reply:
We can probably break this file down in a separate PR. Would be glad to do that.